applications/ChimeraApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -0,0 +1,75 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+// ==============================================================================
+//  ChimeraApplication
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Authors:        Aditya Ghantasala, https://github.com/adityaghantasala
+// 					        Navaneeth K Narayanan
+//					        Rishith Ellath Meethal
+// ==============================================================================
+//
+// External includes
+
+#include "spaces/ublas_space.h"
+#include "linear_solvers/linear_solver.h"
+
+// Project includes
+#include "custom_python/add_custom_utilities_to_python.h"
+#include "custom_utilities/solver_settings.h"
+#include "custom_utilities/fractional_step_settings_for_chimera.h"
+
+//Processes
+namespace Kratos
+{
+
+namespace Python
+{
+
+    // Utility function to transfer the solution step data between the modelparts.
+    // Used in the python file chimera_modelpart_import.py
+    void TransferSolutionStepData(ModelPart& rFromModelPart, ModelPart& rToModelPart)
+    {
+      rToModelPart.SetNodalSolutionStepVariablesList( rFromModelPart.pGetNodalSolutionStepVariablesList() );
+    }
```

Comment:
I found this a bit hacky. Maybe we can slightly modify the `FastTransferBetweenModelPartsProcess` to have an extra option to only transfer the variables container. @loumalouomega what do you think about?

Reply:
Well this I needed to do. Since it is not available in other utilities or processes I did it this way.

Reply:
Ofcourse, when it is available else where I will gladly use it.